src/app/app.js

```diff
@@ -82,6 +82,8 @@ angular
 
         // SVG Polyfill for IE11 @todo lazy load
         window.svg4everybody();
+
+        navigator.registerProtocolHandler("mailto", "https://mail.protonmail.com/?mailtoUrl=%s", "Protonmail");
```

Comment:
maybe add some if for not triggering in dev enviroment? Also, hardcoded URL is bad - but i dont know much about your deployment process and enviroments.